api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -98,7 +98,7 @@
 import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
 import org.springframework.transaction.annotation.Transactional;
 import org.xml.sax.InputSource;
-
+import org.apache.commons.io.FileUtils;
```

Comment:
This should go with the imports above, ideally just before `import org.apache.commons.io.IOUtils;`

Reply:
@ibacher Thanks let me fix it.